backend/src/types.ts

```diff
@@ -220,3 +222,83 @@ export type BuildStatus = {
   status: BUILD_PHASE;
   timestamp?: string;
 };
+
```

Comment:
@dlabaj I think as @maroroman makes progress, this types.ts folder is going to get rather large.  What do you think about a directory with smaller chunks of types?

Reply:
@cfchase That works to break up this file.